internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -262,12 +283,84 @@ func (r *BundleInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		Type:    rukpakv1alpha1.TypeInstalled,
 		Status:  metav1.ConditionTrue,
 		Reason:  rukpakv1alpha1.ReasonInstallationSucceeded,
-		Message: fmt.Sprintf("instantiated bundle %s successfully", bi.Spec.BundleName),
+		Message: fmt.Sprintf("instantiated bundle %s successfully", bundle.GetName()),
 	})
-	bi.Status.InstalledBundleName = bi.Spec.BundleName
+	bi.Status.InstalledBundleName = bundle.GetName()
+
+	if _, err := r.reconcileOldBundles(ctx, oldBundles, bundle); err != nil {
+		meta.SetStatusCondition(&bi.Status.Conditions, metav1.Condition{
+			Type:    rukpakv1alpha1.TypeProgressing,
+			Status:  metav1.ConditionFalse,
```

Comment:
What's your working definition of "Progressing". My initial thought for this would have been Progressing=true, since we're going to try to re-reconcile and ostensibly attempt cleanup again.

Reply:
See https://github.com/operator-framework/rukpak/pull/293#discussion_r865332918 - I ended up deciding to drop this implementation in favor of logging and returning an error. We can iron out the desired UX over time?